### ISSUE TITLE
BUG_02 - Volver a clasificar sin marcar URL

### ISSUE THREAD

**Issue by EnriquePavon73** [zahori-io/zahori-framework#260]
1. Introducir URL en campo URL a escanear. La config utilizada ha sido:
1.1. Versión WCAG: 2.1
1.2. Nivel accesibilidad: AA
1.3. Profundidad de páginas: 3
2. Pulsar en botón "Analizar"
3. Una vez analizado sin elegir ninguna URL pulso sobre clasificar con IA y salta un error

_**Error: Http failure response for https://waaai.panel.es/ia/process_data: 500 INTERNAL SERVER ERROR. SyntaxError: Unexpected token '<', "<!doctype "... is not valid JSON**_

**Comment by EnriquePavon73** [comment]
Comprobamos que ya no salta el error


![Image](https://github.com/zahori-io/zahori-framework/assets/170330241/cd9ef2d7-e66d-4cef-9219-df749d5bccbd)
